components/layout/display_list_builder.rs

```diff
@@ -1072,15 +1072,19 @@ impl FragmentDisplayListBuilding for Fragment {
                                                     style: &ServoComputedValues,
                                                     bounds: &Rect<Au>,
                                                     clip: &ClippingRegion) {
+        use style::values::Either;
+
         let width = style.get_outline().outline_width;
         if width == Au(0) {
             return
         }
 
-        let outline_style = style.get_outline().outline_style;
-        if outline_style == border_style::T::none {
-            return
-        }
+        let outline_style = match style.get_outline().outline_style {
+            // FIXME: handle case when "outline-style: auto". For now, assume
+            //        auto is akin to border_style::T::none.
+            Either::First(_) | Either::Second(border_style::T::none) => return,
```

Comment:
could you use `Either::First(_auto)` here, so it's clear we're not ignoring the style.

Also, let's use `border_style::T::solid` instead of not drawing anything, since it's what Firefox does (which has `outline-style: auto` handling disabled by default).